rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/util/RxBleAdapterWrapper.java

```diff
@@ -38,31 +41,51 @@ public boolean isBluetoothEnabled() {
         return bluetoothAdapter != null && bluetoothAdapter.isEnabled();
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Android Studio shows "redundant suppression" warning for me here

Reply:
Same. But if you remove it then javac warns.